### DISCUSSION TITLE
Creating GH-Pages

### DISCUSSION THREAD

**Discussion by SHADOWELITE7** [community/community#21868] [Other Feature Feedback, Questions, & Ideas]
<p>I have created a repository for my Web Design Portfolio. I was wondering if like when I create it to be a gh branch, does it put it into a pull request or it just overrides the committed files</p>

**Comment by md2perpe** [comment]
<p>Are your committed files in the master branch? Do you want these to be in the gh branch?</p>

**Comment by SHADOWELITE7** [comment]
<p>yes, my files are committed and I would like them to be in the GH Branch</p>
<p><a href="https://github.com/SHADOWELITE7/Web-Design-Portfolio" rel="nofollow noopener">https://github.com/SHADOWELITE7/Web-Design-Portfolio</a></p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/shadowelite7">@shadowelite7</a>,</p>
<p>If you create a new branch called gh-branch from the master branch, it will contain all of the files that were in your master branch.</p>
<p>Additionally, you may be interested in the <a href="/t5/Support-Series/bg-p/SupportSeries/label-name/getting%20started%20with%20github%20pages">Getting Started with GitHub Pages series</a> if you haven’t already read through it.</p>
<p>Thanks!</p>

**Comment by SHADOWELITE7** [comment]
<p>ok thx for the link</p>
